app/pages/AppContainer.js

```diff
@@ -29,9 +33,15 @@ export class UnconnectedAppContainer extends Component {
 UnconnectedAppContainer.propTypes = {
   children: PropTypes.element,
   fetchAuthState: PropTypes.func.isRequired,
+  isAuthFetched: PropTypes.bool.isRequired,
   fetchUnits: PropTypes.func.isRequired,
 };
 
 const actions = { fetchAuthState, fetchUnits };
 
-export default connect(null, actions)(UnconnectedAppContainer);
+const isAuthFetchedSelector = createSelector(
```

Comment:
- [ ] This could be named just `selector` to be consistent with other containers in the project.